src/main/kotlin/org/cascadebot/cascadebot/commands/management/welcome/WelcomeCommand.kt

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2020 CascadeBot. All rights reserved.
+ * Licensed under the MIT license.
+ */
+
+package org.cascadebot.cascadebot.commands.management.welcome
+
+import net.dv8tion.jda.api.entities.Member
+import org.cascadebot.cascadebot.commandmeta.CommandContext
+import org.cascadebot.cascadebot.commandmeta.MainCommand
+import org.cascadebot.cascadebot.commandmeta.Module
+import org.cascadebot.cascadebot.commandmeta.SubCommand
+import org.cascadebot.cascadebot.messaging.MessageType
+import org.cascadebot.cascadebot.messaging.embed
+import org.cascadebot.cascadebot.permissions.CascadePermission
+import org.cascadebot.cascadebot.utils.pagination.Page
+import org.cascadebot.cascadebot.utils.pagination.PageObjects
+import org.cascadebot.cascadebot.utils.placeholders.PlaceholderObjects
+import org.cascadebot.cascadebot.utils.toPercentage
+import org.cascadebot.cascadebot.utils.truncate
+
+class WelcomeCommand : MainCommand() {
+
+    override fun onCommand(sender: Member, context: CommandContext) {
+        if (context.args.isNotEmpty()) {
+            context.uiMessaging.replyUsage()
+            return
+        }
+        val welcomeMessages = context.data.management.greetings.welcomeMessages
+
+        val totalWeight = welcomeMessages.totalWeight
+        val items = welcomeMessages.itemsAndWeighting
+
+        val overviewPage = PageObjects.EmbedPage(embed(MessageType.INFO) {
+            title { name = context.i18n("commands.welcome.messages_title") }
+            field {
+                name = "Message count"
```

Comment:
Lol. (More language)

Reply:
I THOUGHT I GOT THIS ONE OKAY (im sorry)